cheevos-new/cheevos.c

```diff
@@ -380,11 +382,18 @@ static int rcheevos_parse(const char* json)
       lboard->info = rcheevos_locals.patchdata.lboards + j;
       res = rc_lboard_size(lboard->info->mem);
 
+      CHEEVOS_LOG(RCHEEVOS_TAG "rc_lboard_size() = %d\n", res);
```

Comment:
This log message seems unnecessary. We really only care when there's an error, and that's captured on line 394

Reply:
whoops! it's a leftover (used while I was debugging).